### ISSUE TITLE
Setting unknown property: craft\behaviors\CustomFieldBehavior::authorId 

### ISSUE THREAD

**Issue by billymedia** [craftcms/feed-me#845]
### Description
i had a feed running fine for months, then all of a sudden i am getting this error 

`Setting unknown property: craft\behaviors\CustomFieldBehavior::authorId - BaseObject.php: 163. `

The feed is an xml feed with no author related data, i have set it to use the default author. I read that checking the permissions on the compiled_classes folder but this doesnt make a difference.

### Steps to reproduce

1. Create xml feed with no author id, and set default author in feed me mappings screen
2. Run feed

### Additional info

- Craft version: 3.5.12
- PHP version: 7.3.27
- Database driver & version: mysql 5.5.5
- Plugins & versions: 4.3.6

**Comment by adrianjean** [comment]
Hmm — adding my voice to this older issue — I also seeing this problem.

I am importing from a CSV that only has Name, Email — so I set the default author to be my admin account, and set the import setting to "Use Default".

When importing the CSV file, I also get an error similar to the OP.

`Setting unknown property: craft\elements\db\UserQuery::elements.id....`

When I set the author to "Don't Import" — it runs fine, but doesn't attribute the entries to an author.

Again, I'm not importing the author data from the CSV, I'm telling FeedMe to use an author on the system, and in my case an admin author (me).

Craft version: Craft Pro 5.2.4.1
PHP version: 8.3.8
Database driver & version: MySQL 8.0.30
Plugins & versions: 6.1.0
